spec/plans/results.fmf

```diff
@@ -75,6 +79,18 @@ description: |
     may omit all other keys, although tmt plugins will strive to provide
     as many keys as possible.
 
+    .. Duplication warning: after updating the following paragraph, copy it
```

Comment:
Will this be rendered? if yes, we would rather see this as a comment in this file, I do not think it is good idea to render it.

Reply:
No, it's not going to be rendered, it's a ReST comment: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#comments

Reply:
oh my rest, thx

Reply:
What about this part, is it also a comment? Just pointing out that it's a different style.
```
  # Duplication warning: after updating the following examples, copy them into
  # /spec/tests/result.fmf.
  #
  # The goal is to treat results.fmf as the main source of truth, and while
  # it's useful to show custom results examples in test specification as well,
  # let's strive to keep it in sync.
```

Reply:
Both are comments, just different places, different "languages".

* `# Duplication warning` is a YAML comment,
* `.. Duplication warning` is a ReST comment which is part of a multiline string. YAML parser does not treat this structure as a comment, it's part of an opaque blob of characters. Sphinx, on the other hand, needs to be hinted to not render this, done by `..` directive. And Sphinx will never see the YAML comment above, as it's not part of any string consumed by Sphinx.